content/docs/user-guide/project-structure/dvc-files.md

```diff
@@ -76,6 +76,7 @@ The following subfields may be present under `outs` entries:
 | `type`                          | User-assigned type of the data.                                                                                                                                                                                                                                                                                                                       |
 | `labels`                        | User-assigned labels to add to the data.                                                                                                                                                                                                                                                                                                              |
 | `meta`                          | Custom metadata about the data.                                                                                                                                                                                                                                                                                                                       |
+| `push`                          | (Optional) Whether the output file/dir should be pushed to the remote on `dvc push` (`true` by default: outputs are pushed to remotes). `push` only applies to <abbr>cached</abbr> outputs.                                                                                                                                                           |
```

Comment:
Everything is optional except the first field. Also, slight rephrase (similar to `cache` text above):
```suggestion
| `push`                          | Whether or not this  file or directory, when previously <abbr>cached</abbr>, is uploaded to remote storage by `dvc push` (`true` by default). |
```